src/braket/jobs/quantum_job_creation.py

```diff
@@ -161,14 +161,15 @@ def prepare_quantum_job(
     _validate_params(param_datatype_map)
     aws_session = aws_session or AwsSession()
     device_config = DeviceConfig(device)
-    job_name = job_name or _generate_default_job_name(image_uri=image_uri)
+    timestamp = str(int(time.time() * 1000))
```

Comment:
Have we finalized if we want the actual timestamp, or a formatted date/time that the customer would be able to read more easily?

Reply:
Haven't confirmed. Personally I find UTC time not that much more helpful than a timestamp, but happy to to pivot on this. Easy one-line change in the implementation

Reply:
Yes, but I'd rather we settle on it as part of this discussion than kick the conversation down the road. I think it should be a fairly quick decision to make.